Automattic/wp-calypso
# Jetpack: apply correct typeface to Scan and Backup headings

### Changes proposed in this Pull Request

An update in the `FormattedHeader` component required the `brandFont` prop to be passed explicitly so that the Recolata typeface is used. This PR adds this prop to instances of `FormattedHeader` in the _Backup_ and _Scan_ sections.

### Testing instructions

- Download the PR and run Calypso with the `jetpack/features-section` flag enabled.
- For the following sections, check that the top heading has the proper typeface (see screenshot below).
  * With a free plan site, go to _Backup_ and check the title of the Business plan upsell page
  * With a site with Backup enabled, go to _Backup_ and check the title of the page
  * With a site with Scan enabled, go to _Scan_ and check the titles of the _Scanner_ and _History_ page
  * With a Jetpack site with neither Backup nor Scan enabled, go to the _Scan_ and _Backup_ pages and check the titles of the corresponding upsell pages.

_Note: I don't think there's a way to see the Business plan upsell page from the Scan section anymore_

### Screenshots

<img width="346" alt="Screen Shot 2020-06-18 at 2 27 24 PM" src="https://user-images.githubusercontent.com/1620183/85058403-14b3c280-b170-11ea-9976-4044fc3fdd20.png">
